releases/v0.8.0.md

```diff
@@ -0,0 +1,93 @@
+# Release Notes for v0.8.0
+
+Release Date: November 8th, 2023
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+
+* Upstream containerd supported by all deployment types except enclave-cc.
+  * This release includes the Nydus snapshotter (for the first time) to support upstream containerd.
+  * In this release images are still pulled inside the guest.
+  * **Nydus snapshotter requires the following annotation for each pod `io.containerd.cri.runtime-handler: <runtime-class`.**
+  * Nydus snapshotter support is still evolving. See limitations section below for details.
+* A new component, the Confidential Data Hub (CDH) is now deployed inside the guest.
+  * CDH is an evolution of the Attestation Agent that supports advanced features.
+  * CDH supports sealed Kubernetes secrets which are managed by the control plane, but securely unwrapped inside the enclave.
+  * CDH supports connections to both KBS and KMS.
+* New architecture of Attestation Agent and CDH allows a client to deploy multiple KBSes.
+  * One KBS can be used for validating evidence with the Attestation Service while another can provide resources.
+* Pulling from an authenticated registry now uses `imagePullSecrets`.
```

Comment:
I think it's more that
Pulling from an authenticated registry now *requires* `imagePullSecrets` even when pulling on the guest as the nydus-snapshotter need the secret to pull down the metadata. Previously the creds could just be passed in via the attestation-agent's getresources.

Sorry if I wasn't clear in explaining it (via Wainer's update).

Reply:
Ok, make sense.

Reply:
How does that relate to `Handle container registry authentication via userdata` ?

Reply:
Good question. @wainersm can confirm, but I think that line is referring to how we allow the user to pass through the credentials into the peer pod (to be used in the guest pull by image-rs) via userdata/metadata-serviec, which means that they can use a common image without having to custom build it. The technical mechanism of doing it has updated, but the user experience hasn't too much here compared to previous releases, we are now using the agent-config option that Ding added, rather going via the offline_fs_kbc-resource.kbon.

Pre-nydus-snapshotter this was enough to pull authenticated registry images, but as with local hypervisor we now need the imagePullSecrets as well. I hope that helps a bit?